application/src/main/java/org/thingsboard/server/controller/RuleChainController.java

```diff
@@ -326,7 +329,7 @@ public PageData<RuleChain> getRuleChains(
             if (typeStr != null && typeStr.trim().length() > 0) {
                 type = RuleChainType.valueOf(typeStr);
             }
-            return checkNotNull(ruleChainService.findTenantRuleChainsByType(tenantId, type, pageLink));
+            return checkNotNull(ruleChainService.findRuleChainsWithErrorStatistics(tenantId, type, pageLink));
```

Comment:
Let's rename it to **findTenantRuleChainInfosByType**.
If in the future new fields will be added into RuleChainInfo object, then method findRuleChainsWithErrorStatistics is no longer valid and doesn't fully shows real object, that is going to be returned.